settings/installer/ospd/network/isolation/no-isolation.yml

```diff
@@ -0,0 +1,2 @@
+---
+enable: "no"
```

Comment:
too general, remember, when the whole file is merged how would you know this is related to no isolation?
I would keep it as: "installer.network.isolation.enable: no"